backend/pkg/logger/data/datalog.go

```diff
@@ -0,0 +1,33 @@
+package datasublogger
+
+import (
+	"encoding/csv"
+	"os"
+	"time"
+
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+)
+
+type CSVSublogger struct {
+	file   *os.File
```

Comment:
```suggestion
	output   io.WriteCloser
```
use interface instead so that you can change it easily